frontend/src/components/landingPage/SearchPart.vue

```diff
@@ -55,38 +68,100 @@ export default {
   },
   data() {
     return {
-      savedQuery: "",
-      readyToSearchQuery: "",
-      finalQueryToReset: false,
+      savedQuery: "", //query from queryMenu will be saved here
+      readyToSearchQuery: "", //finalQuery from the query component will be saved here
       headers: [
-        { text: "ArtifactId", align: "left", value: "metadata.artifactId" },
-        { text: "GroupId", value: "metadata.groupId" },
-        { text: "Source", value: "metadata.source" },
-        { text: "Version", value: "metadata.version" }
+        { text: "ArtifactId", align: "left", value: "metadata.artifactId", width: "10%" },
+        { text: "GroupId", value: "metadata.groupId", width: "10%" },

```

Comment:
Could you reorder groupId and artifactId so groupId comes first? That's a bit more natural for people knowing Maven.

Reply:
Done